### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Illia-L**
- [DEMO LINK](https://Illia-L.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Illia-L.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,49 @@
...
+          alt="logo Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a class="nav__link is-active" href="#">Apple</a></li>
+          <li><a class="nav__link" href="#">Samsung</a></li>
+          <li><a class="nav__link" href="#">Smartphones</a></li>
+          <li>
+            <a class="nav__link" href="#"
```
Comment (YegorKorenkov) [inline src/index.html:39]: If number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag

File: src/style.css [hunk 2]
```diff
@@ -1 +1,81 @@
...
+div,
+ul,
+li,
+a {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  max-width: 1200px;
```
Comment (YegorKorenkov) [inline src/style.css:15]: Please remove it, header should be fullwidth

File: src/style.css [hunk 3]
```diff
@@ -1 +1,81 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```
Comment (YegorKorenkov) [inline src/style.css:24]: Please add padding for header Istead of using left/right margin for logo and nav. Should be smth like this:
image

File: src/style.css [hunk 4]
```diff
@@ -1 +1,81 @@
...
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
+  margin-right: 20px;
+  font-family: "Roboto", Arial, sans-serif;
```
Comment (YegorKorenkov) [inline src/style.css:46]: Please move to html tag selector

File: src/style.css [hunk 5]
```diff
@@ -1 +1,81 @@
...
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:link,
+.nav__link:visited {
+  color: #000;
```
Comment (YegorKorenkov) [inline src/style.css:55]: if you add this color rule general selector this should work the same but there will be less unnecessary selectors

File: src/style.css [hunk 6]
```diff
@@ -1 +1,81 @@
...
+}
+
+.nav__link:hover,
+.nav__link:active {
+  color: var(--link-hover-active);
```
Comment (YegorKorenkov) [inline src/style.css:60]: the same

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 6 above)
Comment (Illia-L) [reply src/style.css:60]: this one is not clear for me. Please more hints

File: src/style.css [hunk 7]
```diff
@@ -1 +1,75 @@
...
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
```
Comment (polosanya) [inline src/style.css:24]: It would be great to create variable for all main values and colors

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done!

File: src/style.css (hunk 7 above)
Comment (Illia-L) [reply src/style.css:24]: Not sure I can figure out main values in this task. Would you please give me either refference or list of main values for this task

Comment (TarasHoliuk) [reply src/style.css:24]: That might be smth like `--logo-size: 40px`, but not critical IMO

File: src/index.html [hunk 8]
```diff
@@ -2,13 +2,54 @@
...
+      <a class="logo-link" href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a class="nav__link is-active" href="#">Apple</a></li>
```
Comment (TarasHoliuk) [inline src/index.html:35]:
```suggestion
          <li>
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
```
preferred formatting for nested tags (FYI)

**Review by TarasHoliuk — APPROVED** [review]
Task approved :heavy_check_mark: 
No need to re-request a review. Comments I've left are FYI

**Pull request closed without merging**